zooniverse/front-end-monorepo
# Homepage: Build user stats preview in dashboard

- [x] Add fetching functions in DashboardContainer for user's classification numbers per current week and all time.

## Package
lib-user

## Linked Issue and/or Talk Post
Follows: https://github.com/zooniverse/front-end-monorepo/pull/6117
Figma: [Signed-in homepage](https://www.figma.com/design/qbqbmR3t5XV6eKcpuRj7mG/Group-Stats?node-id=1981-11239&t=FXkBJkbFeX6RQhvA-4)

## Describe your changes
- Completed all components for a user's Dashboard. The styling is now complete according to the Figma design linked above.

⚠️  This PR does not include robust Story components or unit tests for the Dashboard, but I'm speeding along to get this user home page built before internal team testing.

## How to Review

- View the Dashboard stories. Note that because TabsContainer utilizes `useStats()`, and it's not mocked in the Dashboard stories, actual stats numbers will not appear.
- Run app-root locally with `yarn build` and `yarn start` and visit https://local.zooniverse.org:3000/?env=production
- Run lib-user locally with `yarn dev` and visit: https://local.zooniverse.org:8080


# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
